internal/c/parts/audio/out/src.c

```diff
@@ -1149,15 +1149,15 @@ void sub__sndclose(int32 handle) {
 
 //"macros"
 
-void sub__sndplaycopy(int32 handle, double volume, int32 passed) {
+void sub__sndplaycopy(int32 handle, double volume, double x, double y, double z, int32 passed) {

```

Comment:
Yeah I agree with not bothering with adding support in the OpenAL backend. We should create a plan for getting rid of it, I'll look into how we can organize that.